MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,33 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.95.0 ➞ v0.96.0
+
+### *(breaking change)* resource_monitor resource
+Removed fields:
+- `set_for_account` (will be settable on account resource, right now, the preferred way is to set it through unsafe_execute resource)
+- `warehouses` (can be set on warehouse resource, optionally through unsafe_execute resource)
```

Comment:
it can't be set in unsafe execute if warehouses are managed by terraform because warehouse resource will remove it if not set in its config.

Reply:
True, changed